CMakeLists.txt

```diff
@@ -427,10 +420,8 @@ add_library (${LIB_NAME}
 target_compile_definitions(${LIB_NAME} PUBLIC "$<$<CONFIG:Debug>:ARABICA_DEBUG>")
 target_compile_definitions(${LIB_NAME} PUBLIC ARABICA_NOT_USE_PRAGMA_LINKER_OPTIONS)
 target_include_directories(${LIB_NAME} PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/include>)
-target_include_directories(${LIB_NAME} PUBLIC ${ADDITIONAL_INC} ${Boost_INCLUDE_DIRS})
 target_include_directories(${LIB_NAME} PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_LIST_DIR}/include>)
-# link backend
-target_link_libraries(${LIB_NAME} ${ADDITIONAL_LIB})
+target_include_directories(${LIB_NAME} PUBLIC $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/${PROJECT_NAME}>)

```

Comment:
Usually, it's just `include`, i.e. it should be:
```cmake
target_include_directories(${LIB_NAME} PUBLIC $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>)
```

Reply:
Unfortunately a lot of the headers internally refer to each other without using the `arabica` prefix. E.g. in `arabica/SAX/XMLReader.hpp` we can find the following:
```
#include <SAX/ArabicaConfig.hpp>
```

I have not been able to get this to work without some sort of equivalent to the above. But we can definitely make sure that we only use `INCLUDES DESTINATION include/${PROJECT_NAME}` instead of the construction seen above.

Reply:
> E.g. in arabica/SAX/XMLReader.hpp we can find the following

In my opinion, that's the wrong approach, but we can't do anything with it. It should be improved upstream.

> But we can definitely make sure that we only use INCLUDES DESTINATION include/${PROJECT_NAME}

Okay, that's fine in this case.